client/index.js

```diff
@@ -182,25 +182,19 @@ exports.focus = function (data) {
  * @name get
  * @function
  * @param {Object} data The data object containing:
+ *
+ *  - `callback` (Function): An optional callback function.
  */
 exports.get = function (data, stream) {
     var self = this;
     var value = this.editor.getValue();
 
     // fire callback (if exists)
-    if (data) {
-        var callback = data.callback || function () {};
-        callback(value);
+    if (data && typeof data.callback === "function") {
+        data.callback(value);
     }
 
-    if (stream) {
```

Comment:
@danandrei Why is this needed? This was causing the `set` function to be called twice.